ecommerce/src/main/kotlin/io/hhplus/ecommerce/cart/application/handler/CartEventHandler.kt

```
@@ -24,17 +31,39 @@ class CartEventHandler(
     private val json = Json { ignoreUnknownKeys = true }
 
     override fun supportedEventTypes(): List<String> {
-        return listOf(EventRegistry.EventTypes.PAYMENT_COMPLETED)
+        return listOf(
+            EventRegistry.EventTypes.ORDER_CREATED,
+            EventRegistry.EventTypes.ORDER_CANCELLED
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)